pkgs/applications/editors/vim/macvim.nix

```diff
@@ -217,5 +217,31 @@ stdenv.mkDerivation (finalAttrs: {
     hydraPlatforms = [ ]; # hydra can't build this as long as we rely on Xcode and sandboxProfile
     # Needs updating to a newer MacVim for Python and Ruby version support
     broken = true;
+    knownVulnerabilities = [
+      "CVE-2023-46246"
+      "CVE-2023-48231"
+      "CVE-2023-48232"
+      "CVE-2023-48233"
+      "CVE-2023-48234"
+      "CVE-2023-48235"
+      "CVE-2023-48236"
+      "CVE-2023-48237"
+      "CVE-2023-48706"
+      "CVE-2023-5344"
+      "CVE-2023-5441"
+      "CVE-2023-5535"
+      "CVE-2024-22667"
+      "CVE-2024-41957"
+      "CVE-2024-41965"
+      "CVE-2024-43374"
+      "CVE-2024-47814"
+      "CVE-2025-1215"
+      "CVE-2025-22134"
+      "CVE-2025-24014"
+      "CVE-2025-26603"
+      "CVE-2025-29768"
+      "CVE-2025-53905"
+      "CVE-2025-53906"
```

Comment:
Where did this list come from? Everything up to `CVE-2024-22667` is about Vim 9.0.x versions, and MacVim r179 is Vim 9.1. The ones starting with `CVE-2024-41957` are presumably valid, since that one is about Vim 9.1.x.

Reply:
It’s from https://nvd.nist.gov/vuln/search#/nvd/home?cpeFilterMode=cpe&cpeName=cpe:2.3:a:vim:vim:9.0.1897:*:*:*:*:*:*:*&resultType=records; I think I mixed up the base Vim version with the previous one. The correct list is presumably https://nvd.nist.gov/vuln/search#/nvd/home?cpeFilterMode=cpe&cpeName=cpe:2.3:a:vim:vim:9.1.0000:*:*:*:*:*:*:*&resultType=records, which matches CVE-2024-41957 as the starting point. Sorry about that. Fix is https://github.com/NixOS/nixpkgs/pull/454720.

Reply:
CPE are always a pleasure to work with :weary: